tests/e2e/upgradecluster/Vagrantfile

```diff
@@ -91,7 +91,17 @@ end
 
 def getDBType(role, role_num, vm)
 
-  if EXTERNAL_DB == "mysql"
+  if EXTERNAL_DB == "mariadb"
+    if role.include?("server") && role_num == 0
+      dockerInstall(vm)
+      vm.provision "Start mysql", inline: "docker run -d -p 3306:3306 --name #{EXTERNAL_DB} -e MYSQL_ROOT_PASSWORD=e2e mysql:5.7"
+      vm.provision "shell", inline: "echo \"Wait for mysql to startup\"; sleep 10"
+      return "datastore-endpoint: 'mysql://root:e2e@tcp(#{NETWORK_PREFIX}.100:3306)/k3s'"
+    elsif role.include?("server") && role_num != 0
+      return "datastore-endpoint: 'mysql://root:e2e@tcp(#{NETWORK_PREFIX}.100:3306)/k3s'"
+    end
+
```

Comment:
```suggestion
    if role.include?("server") && role_num == 0
      dockerInstall(vm)
      vm.provision "Start mariaDB", inline: "docker run -d -p 3306:3306 --name #{EXTERNAL_DB} -e MARIADB_ROOT_PASSWORD=e2e mariadb:11"
      vm.provision "shell", inline: "echo \"Wait for mariaDB to startup\"; sleep 10"
      return "datastore-endpoint: 'mysql://root:e2e@tcp(#{NETWORK_PREFIX}.100:3306)/k3s'"
    elsif role.include?("server") && role_num != 0
      return "datastore-endpoint: 'mysql://root:e2e@tcp(#{NETWORK_PREFIX}.100:3306)/k3s'"
    end
```

Reply:
This should get you up and running. The same changes will need to be applies to the validatecluster test. 

Additionally, please sign all your commits to comply with the DCO.

Reply:
@dereknola, thanks for help. I believe the commits now comply with DCO